### DISCUSSION TITLE
Is orientdb still active?

### DISCUSSION THREAD

**Discussion by mobius1983** [orientechnologies/orientdb#10243] [General] [answered]
I have a query I want to start using this orientdb database but I am worried that at some point it will be deprecated, because I saw the last release was in 2020, thank you

**Comment by tglman** [comment, marked as answer]
https://github.com/orientechnologies/orientdb/releases

**Reply by mobius1983** [reply]
Thank you very much, I went to https://orientdb.org/release-notes from the website and saw the releases, the last one was from 2020
